core/scripts.py

```diff
@@ -2758,6 +2766,166 @@ def suppress_work(self, library: Library, identifier: Identifier) -> None:
         self._db.commit()
 
 
+class GenerateInventoryReports(Script):
+    """Generate inventory reports from queued report tasks"""
+
+    DATA_SOURCES = [
```

Comment:
Since datasource is just a free form text input in the OPDS importers, I don't think hard coding the data sources here like this will accomplish what we are hoping for. Maybe this needs to be configurable, or you need to select the collection when you are generating the report. Otherwise we will be relying on the CMs to be setup with these exact strings set as the datasource.